docs/csharp/whats-new/csharp-version-history.md

```diff
@@ -225,19 +243,19 @@ C# 7.2 added several small language features:
 - Add the `in` modifier on parameters, to specify that an argument is passed by reference but not modified by the called method.
 - Use the `ref readonly` modifier on method returns, to indicate that a method returns its value by reference but doesn't allow writes to that object.
 - Declare `ref struct` types, to indicate that a struct type accesses managed memory directly and must always be stack allocated.
-- Use additional generic constraints.
-- [Non-trailing named arguments](../programming-guide/classes-and-structs/named-and-optional-arguments.md)
-  - Named arguments can be followed by positional arguments.
-- Leading underscores in numeric literals
+- Use more generic constraints.
```

Comment:
I like the original version better (even though Acrolinx doesn't like the word 'additional'). It can be read now as that the constraints are "more generic".